### ISSUE TITLE
NTR: Fetal methamphetamine exposure

### ISSUE THREAD

**Issue by peupeubangbang** [obophenotype/human-phenotype-ontology#10601]
**Preferred term label:**
Fetal methamphetamine exposure

**Synonyms**
fetal meth exposure
antenatal methamphetamine exposure
antenatal meth exposure
prenatal methamphetamine exposure
prenatal meth exposure
methamphetamine use during pregnancy
meth use during pregnancy
methamphetamine use in pregnancy
meth use in pregnancy
antenatal methamphetamine use
antenatal meth use
prenatal methamphetamine use
prenatal meth use

**Definition (free text, please give PubMed ID)**
A history of exposure during pregnancy (including during the period of conception) of the mother of a fetus or child to methamphetamine.
PMID:26212684

**Parent term (use hpo.jax.org/app)**
Fetal exposure history HP:0034756

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@peupeubangbang 
thanks!

Adding new term: 
- ID: HP:6000990
- label: Fetal methamphetamine exposure
- parents: [Fetal exposure history (HP:0034756)](https://hpo.jax.org/app/browse/term/HP:0034756)
- pmid: PMID:26212684
- def: A history of exposure during pregnancy (including during the period of conception) of the mother of a fetus or child to methamphetamine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10601
- synonyms: Antenatal methamphetamine exposure(NONE)|Prenatal methamphetamine use(NONE)|Fetal meth exposure(NONE)|Antenatal methamphetamine use(NONE)|Prenatal methamphetamine exposure(NONE)|Methamphetamine use during pregnancy(NONE)
